README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Hello, my name is Sophia Egrari
+
+I just finished my second year at UChicago as a **sociology** major and am interested in possibly completing the data science minor, as well, and am hoping that CFSS will give me a solid introduction to basic programming skills and data visualization!
+
+#### A little bit about me: 
+* I am from Seattle, Washington 
+* In my free time, I love to draw, play guitar, and read a LOT
+* Along with taking CFSS and working as a research assistant, I will also be writing for a blog this summer that advocate for mental health
+>you can check it out at https://www.localoptimist.com
```

Comment:
Don't forget you can also use `[]()` syntax to embed URLs in a document with clickable text